static/js/user_groups_settings_ui.js

```diff
@@ -133,6 +133,28 @@ export function add_group_to_table(group) {
     }
 }
 
+export function update_group(group_id) {
+    const group = user_groups.get_user_group_from_id(group_id);
+    if (!group) {
+        /*
+            We ignore cases where this is called for groups that do not
+            exist or might be deleted when this event was received.
+        */
+        return;
```

Comment:
Is this a case that we expect to actually see? If not, maybe a `blueslip.error` would be appropriate.

Reply:
While adding this I was thinking about the possible race where one user deletes the group and the other updates the group info. But thinking more carefully, it is not possible to get update event for a very recently deleted group due to increasing nature of event ids. So I have removed this check and calls for such non-existent groups would report `blueslip.error`.

Reply:
That sort of out-of-order event thing should be very rare, but is possible if the server is having trouble -- event IDs are allocated in a way that is somewhat independent of the transaction ordering. 

That said, I think doing nothing beyond a `blueslip.error` in the situation that we see such an event editing a deleted group is likely correct.